src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/IssueFurtherEvidenceHandlerFunctionalTest.java

```diff
@@ -2,15 +2,26 @@
 
 import static org.junit.Assert.assertEquals;
 import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.ISSUE_FURTHER_EVIDENCE;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.VALID_APPEAL_CREATED;
 
 import java.io.IOException;
 import java.util.List;
 import org.assertj.core.api.Assertions;
+import org.junit.Before;
 import org.junit.Test;
 import uk.gov.hmcts.reform.sscs.ccd.domain.*;
 
 public class IssueFurtherEvidenceHandlerFunctionalTest extends AbstractFunctionalTest {
 
+    @Before
+    public void slowDownTests() throws InterruptedException {
+        // we have a problem with functional tests failing randomly. It was noticed that there are frequent connection failures,
+        //  so maybe the tests run before the corresponding services are spun up. This method introduces a delay before tests
+        //  to try and give the environment a bit more time to cope.
+
+        Thread.sleep(5000);
+    }
```

Comment:
A dedicated note is required on this one.
Normally you would want to add the `@RepeatIfExceptionsTest` annotation to your tests. However in this case it was noticed that the error was very variable (sometimes a 4xx, sometimes 5xx, sometimes a timeout). This weird variation in the error didn't seem consistent with the type of scenarios that `@RepeatIfExceptionsTest` would solve. In particular, repeating a test as a result of an overload would just add additional stress to the system.

As a result instead of using that annotation it was decided, for this specific case, to use this ugly contraption. The annotation may also have fixed the issue and might become better in case more tests are added to this suite, but for now this solution appears to be more viable.

Of course if we resume having problems on these functional tests, this code might need to be reviewed and removed.